docs/02-guides/server-side-events/create-an-integration.mdx

```diff
@@ -7,7 +7,135 @@ sidebar_position: 1
+}
+```
+
+In this class, we have four entities :
+- `compatibleEvents`: this indicates which events the integration will handle, it can be `*` to handle all events or an array of string that contains compatible events.
+- `webhookId`: this is the id of WebhookCool.
+- `constructor`: it will allow us to implement logic when class is created, here, we will store the webhookId of WebhookCool into our class.
+- `handle`: this is the method that will be called when an event is received.
+
+For your integrations, only `compatibleEvents` and `handle` are required, the `webhookId` and the constructor is an example of custom method and properties that you can implement into your integrations.
+
+## Write specific code for your integration
+
+All the event processing part will be done in the `handle` method. So we need to write all the code to send event to external services here.
+
+Since Webhook.cool is a basic services, we ca use a simple fetch to send the event :
+
+```ts title="app/server-evets/integrations/CoolWebhookIntegration.ts"
+import type { Integration } from "@front-commerce/core/server-events/integrations";
+
+export default class CoolWehbookIntegration implements Integration {
+  compatibleEvents: string[] | "*" = "*";
+
+  webhookId: string;
+
+  constructor(webhookId: string) {
+    this.webhookId = webhookId;
+  }
+
+  handle(messageId: string, payload: any) {
+    // highlight-start
+    fetch(`https://${this.webhookId}.webhook.cool`, {
+      method: "POST",
+      body: JSON.stringify(payload),
+    });
+    // highlight-end
+  }
+}
+```
+Remark that `webhookId` is used into the fetch called URL.
+
+:::info
+
+Here, we are posting the raw event details to the webhook, but feel free to manipulate it's data freely to adapt it to your specific integration.
+
+You'll find in the [FAQ](/docs/3.x/guides/server-side-events/faq#what-are-the-data-returned-by-the-server-events) the data format for the payload object
 
-- Create an integration
 :::
+
+## Register your integration
+
+Now that our integration is created, we have to register it in Front-Commerce.
+
+To do this, go to the `app/config/serverEvents.ts` file and register your new integration like this :
+
+```ts title="app/config/serverEvents.ts"
+import { ConsoleIntegration } from "@front-commerce/core/server-events/integrations";
+// highlight-next-line
+import CoolWehbookIntegration from "../server-events/integrations/CoolWehbookIntegration";
+
+export default {
+  redis: {
+    host:
+      process.env.FRONT_COMMERCE_SERVER_EVENT_REDIS_HOST ||
+      process.env.FRONT_COMMERCE_CLOUD_REDIS_HOST ||
+      process.env.FRONT_COMMERCE_REDIS_HOST ||
+      "127.0.0.1",
+    port:
+      process.env.FRONT_COMMERCE_SERVER_EVENT_REDIS_PORT ||
+      process.env.FRONT_COMMERCE_CLOUD_REDIS_PORT ||
+      process.env.FRONT_COMMERCE_REDIS_PORT ||
+      6379,
+  },
+  integrations: [
+    new ConsoleIntegration(),
+    // highlight-next-line
+    new CoolWehbookIntegration("XXXXXXXXXXXXXX"),
```

Comment:
Use the webhook id from your image, `tight-kitchen-57` this removes any confusion.
```suggestion
    new CoolWehbookIntegration("tight-kitchen-57"),
```

Reply:
As you explain this lower in the info section, you can decide if you keep it like this, but then add the `tight-kitchen-57` into the info section, or if you replace it all together